### ISSUE TITLE
Alignment of column height

### ISSUE THREAD

**Issue by theryankelly** [codeisland/RI-Budget-Sankey#2]
Would like to have three columns relatively same height because they are all the same total amount of money.

Is it possible to align all three columns using padding/white space?  The padding that is added to the third column creates an illusion that the funds are increasing when the total is the same for all three columns.  Fixing the height seems to lose many of the smaller categories.

**Comment by nelsonri72** [comment]
I agree.  It was an issue already brought up.  Perhaps it may default to no padding to appear equal but when selected it may be padded and expand?

**Comment by jamesrhaley** [comment]
This particular issue is how the plug in calculates the spacing.  This is
pretty much a feature addition to that plugin that would be required.  The
plug in can be found here.

https://github.com/d3/d3-plugins/tree/master/sankey

While I was planning on overhauling this plugin after I finished some
courses,  If there was someone in the Code for America world that wished
to contribute I would be happy to collaborate. I am familiar with the
section of the code that would need to be changed.  We could fork it and go
from there.

**Comment by ghost** [comment]
Regardless of whatever fix we eventually work out, I think we need to add some explanatory text IMMEDIATELY. I think a simple sentence should do it. Please check out the Github repo link issue where I will pick up this thread.
